frontend/src/scenes/session-recordings/player/snapshot-processing/process-all-snapshots.ts

```diff
@@ -340,15 +373,22 @@ export const parseEncodedSnapshots = async (
             }
 
             return snapshotData.flatMap((d: unknown) => {
-                const snap = coerceToEventWithTime(d, sessionId)
+                const snapResult = coerceToEventWithTime(d, sessionId, hasSeenAnyFullSnapshot, isMobileSnapshots)
```

Comment:
i meant to literally move this to the process all snapshots
right now coerceToEventWithTime is doing too much (already even)
and we should have one place where we're doing this processing

let me do the change to see if i'm being stupid... hang on